gtk/MakeDialog.cc

```diff
@@ -379,8 +379,8 @@ void MakeDialog::Impl::updatePiecesLabel()
 
 void MakeDialog::Impl::configurePieceSizeScale()
 {
-    // the below lower & upper bounds would allow piece size selection between approx 1KiB - 16MiB
-    auto adjustment = Gtk::Adjustment::create(log2(builder_->pieceSize()), 10, 24, 1.0, 1.0);
+    // the below lower & upper bounds would allow piece size selection between approx 1KiB - 64MiB
```

Comment:
Is this change in line with other clients and CLI utils?

P.S. Not a fan of changing something (seemingly unrelated) in GTK client while the PR title is prefixed with "Qt:".

Reply:
Yes, that's fair. I did wonder whether this was wise to stuff it into only a partially related change set. It's mainly as the change is so small and it brings the GTK limit into line with the Qt based one. This could be applied separately if it's a big no no.